.github/ISSUE_TEMPLATE/propose-an-analysis-template.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Propose an analysis
+about: Use this issue template to propose a new analysis
+title: 'Proposed Analysis:'
+labels: proposed analysis
+assignees: ''
+
+---
+
+<!--Hi there! Please take a moment to fill out the template below.-->
+
+<!-- Check all those that apply or remove this section if it is not applicable.-->
+
+## What are the scientific goals of the analysis?
+
+
+
+
+## What methods and tools will be used to accomplish the scientific goals?
+
+
+
+
+## Add relevant scientific literature that relates to this analysis
+
+
+
+
+## What input data are required for this analysis?
+
+
+
+
+### How long do you expect is needed to complete the analysis? Will it be a multi-step analysis?
```

Comment:
Maybe separate these two questions into different lines. Also, style-wise I feel it would look better for all the questions to have the same size text and still have lines separating the questions like with the questions above

Reply:
This has been now addressed here 3850defc876446372cbf5f595267764fb26a5b75